src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -205,11 +205,11 @@ public async Task<ImmutableArray<CodeFixCollection>> GetFixesAsync(Document docu
             if (result.Count > 0 && _fixerPriorityMap.TryGetValue(document.Project.Language, out var fixersForLanguage))
             {
                 // sort the result to the order defined by the fixers
-                ImmutableDictionary<CodeFixProvider, int> priorityMap = fixersForLanguage.Value;
+                var priorityMap = fixersForLanguage.Value;
                 result.Sort((d1, d2) => GetValue(d1).CompareTo(GetValue(d2)));
 
                 int GetValue(CodeFixCollection c)
-                    => priorityMap.TryGetValue((CodeFixProvider)c.Provider, out var value) ? value : int.MaxValue;
+                    => priorityMap!.TryGetValue((CodeFixProvider)c.Provider, out var value) ? value : int.MaxValue;
```

Comment:
don't love this.

Reply:
It seems unfortunate that nullable warning is generated here, when the declaration explicitly initializes it to a non-null value. I can revert back the declaration to specify explicit type name and suppress "use var" diagnostic on that instance.